Documentation/Storage-Configuration/NFS/nfs-security.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Security
+---
+
+## User ID mapping
+User ID mapping allows the NFS server to map connected NFS client IDs to a different user domain,
+allowing NFS clients to be associated with a particular user in your organization. For example,
+users stored in LDAP can be associated with NFS users and vice versa.
+
```

Comment:
In addition to this overview, I realized I'd ben using "ID **management**" when I should have been using "ID **mapping**".